app/viewmodels/package.scala

```diff
@@ -113,6 +113,15 @@ package object viewmodels {
       Seq(headOption, lastOption).flatten.mkString(separator)
     }
 
+    def firstAndLastRange(separator: String = "..."): String = {
```

Comment:
Just trying to decipher the difference between this and `firstAndLast`...

```
value match {
        case head :: _ if value.size == 1 => (Some(head), value.lastOption)
}
```

So this will always resolve to `(Some, None)`

But `case head :: tail => (Some(head), tail.lastOption)` would also resolve to `(Some, None)` no? Happy to chat about it 👍

Reply:
What you've said makes sense! Could consider changing the first case to

`case head :: Nil => (Some(head), Some(head))` to really emphasise the behaviour